src/main/java/com/lineate/mdzinarishvili/reminderapp/controllers/ReminderController.java

```diff
@@ -1,17 +1,46 @@
 package com.lineate.mdzinarishvili.reminderapp.controllers;
 
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.lineate.mdzinarishvili.reminderapp.models.Reminder;
+import com.lineate.mdzinarishvili.reminderapp.services.ReminderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
+@RequestMapping(path = "reminders")
+
 public class ReminderController {
+    private final ReminderService reminderService;
+

```

Comment:
You don't need this constructor here. Spring will inject all the dependencies itself. It's about controllers only